app/services/create_account.rb

```diff
@@ -41,6 +42,10 @@ def create_redis_namespace
     CreateRedisNamespaceJob.perform_later(account)
   end
 
+  def create_default_admin_set
+    CreateDefaultAdminSetJob.perform_later(account)
```

Comment:
I think this is going to have to happen with `Operation`s.  Where we make some sort of composite operation for Solr + Fedora.  When both of those jobs have marked their part as done (last success), then the AdminSet job could be enqueued.